packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts

```diff
@@ -55,6 +54,14 @@ import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
 import { LanguageService } from '@theia/core/lib/browser/language-service';
 import { ThemeIcon } from '@theia/monaco-editor-core/esm/vs/base/common/themables';
 
+// The enum export is missing from `vscode-textmate@9.2.0`
+const enum StandardTokenType {
+    Other = 0,
+    Comment = 1,
+    String = 2,
+    RegEx = 3
+}
+
```

Comment:
The export still exists, see [here](https://github.com/microsoft/vscode-textmate/blob/f0f2a44b4c055b0e0fa3869e5de338efda5b7cf1/src/encodedTokenAttributes.ts#L159-L165).

The change to this file should be reverted.

Reply:
> Edit: My mistake. The export exists but it's not part of the release build.
> 
> However it's also defined here `@theia/monaco-editor-core/esm/vs/editor/common/encodedTokenAttributes`, so we might want to use that one? Not sure. What do you think?

I can provide an update, but it does not matter; they're the same. Please let me know.

I think it's a bug in the vscode-textmate API and could be the side-effect of an API extractor in https://github.com/microsoft/vscode-textmate/commit/62fcd6fd4b150a570bded80b6de9bc74bff3a1b1.

But I am just guessing what has caused the API break; they did not create a tag:

> * [microsoft/vscode-textmate@v9.1.0...v9.2.0](https://github.com/microsoft/vscode-textmate/compare/v9.1.0...v9.2.0) (there is no `v9.0.0` tag!)